hrjobcontract/js/test/services/contract-details.service.spec.js

```diff
@@ -2,22 +2,21 @@
 
 define([
   'mocks/data/contract.data',
-  'job-contract/modules/job-contract.module'
+  'job-contract/job-contract.module'
 ], function (MockContract) {
   'use strict';
 
   describe('contractDetailsService', function () {
     var $httpBackend, $rootScope, contractDetailsService;
 
-    beforeEach(module('job-contract'));
+    beforeEach(module('job-contract', 'job-contract.templates'));
     beforeEach(inject(function (_contractDetailsService_, _$httpBackend_, _$rootScope_) {
       contractDetailsService = _contractDetailsService_;
       $httpBackend = _$httpBackend_;
       $rootScope = _$rootScope_;
 
       $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond(200);
       $httpBackend.whenGET(/action=get&entity=HRJobDetails/).respond(MockContract.contract);
-      $httpBackend.whenGET(/views.*/).respond({});
```

Comment:
This deletion is not mentioned in the PR description, why did we need to mock the backend for `views` and now not?

Reply:
The reason why the calls to `views/` were mocked was because the `xyz.templates` module was not used, though available. This module is created on the fly by karma exactly to avoid http requests to templates
```js
// Used to transform angular templates in JS strings
preprocessors: (function (obj) {
  obj[extPath + 'js/src/job-contract/**/*.html'] = ['ng-html2js'];
  return obj;
})({}),
```

Now that the `views/` folder is gone, and we are using the `xyz.templates` module in the test files, we don't need to mock the calls anymore